docs/book/src/tasks/experimental-features/machine-pools.md

```diff
@@ -26,3 +26,15 @@ Although MachinePools provide a similar feature to MachineDeployments, MachinePo
 | Each MachinePool corresponds 1:1 with an associated InfraMachinePool.                                                                                               | Each MachineDeployment includes a MachineSet, and for each replica, it creates a Machine and InfraMachine.                             |
 | Each MachinePool requires only a single BootstrapConfig.                                                                                                            | Each MachineDeployment uses an InfraMachineTemplate and a BootstrapConfigTemplate, and each Machine requires a unique BootstrapConfig. |
 | Maintains a list of instances in the `providerIDList` field in the MachinePool spec. This list is populated based on the response from the infrastructure provider. | Maintains a list of instances through the Machine resources owned by the MachineSet.                                                   |
+
+## MachinePool provider implementations
+
+The following Cluster API infrastructure providers have implemented support for MachinePools:
+
+| Provider | Implementations | Status | Documentation |
+| --- | --- | --- | --- |
+| Amazon | `AWSManagedMachinePool` / `AWSMachinePool` | Implemented, MachinePoolMachines supported | https://cluster-api-aws.sigs.k8s.io/topics/machinepools.html|
+| Azure | `AzureASOManagedMachinePool` / `AzureManagedMachinePool` / `AzureMachinePool` | Implemented, MachinePoolMachines supported | https://capz.sigs.k8s.io/self-managed/machinepools |
+| Oracle | `OCIManagedMachinePool` / `OCIMachinePool` | Implemented | https://oracle.github.io/cluster-api-provider-oci/managed/managedcluster.html |
```

Comment:
OCI provider appears to have support of MachinePoolMachines as `OCIMachinePoolMachine`.

https://github.com/oracle/cluster-api-provider-oci/pull/304